airgun/settings.py

```diff
@@ -38,9 +44,26 @@ class SeleniumSettings(object):
 
     def __init__(self):
         self.browser = None
+        self.saucelabs_key = None
+        self.saucelabs_user = None
+        self.screenshots_path = None
         self.webdriver = None
         self.webdriver_binary = None
-        self.screenshots_path = None
+        self.webdriver_desired_capabilities = None
+
+
+class WebdriverCapabilitiesSettings(object):
+
+    def __init__(self):
+        self.platform = None
+        self.version = None
+        self.maxDuration = None
+        self.idleTimeout = None
+        self.seleniumVersion = None
+        self.build = None
+        self.screenResolution = None
+        self.tunnelIdentifier = None
+        self.tags = None
```

Comment:
For constructing such data holder objects, the `attrs` library is ideal :)

Reply:
Thanks, will think about it when i finally find some time to work on settings reader :) Current implementation is very simplistic and maybe it's worth to move from configparser to something more powerful, not to mention these objects. On the other hand - it works, and simple is better than complex, isn't it? :D

Reply:
`attrs` is simple :stuck_out_tongue_closed_eyes:

```python
import attr

@attr.s
class WebdriverCapabilitiesSettings(object):
    platform = attr.ib(default=None)
    version = attr.bi(default=None)
    # ...
```

You are then spared of writing the init, you can specify which attrs can be initialized via init, you can add types ... all in the definition :)